index.js

```diff
@@ -277,6 +277,21 @@
     return Nothing();
   });
 
+  //. pluck :: String -> [Object] -> [Maybe *]
```

Comment:
Yes, I like that idea. The `*` would provide a visual link between the type of the second argument and the return type.